scripts/BilbyCustomFunctions_Reduction.py

```diff
@@ -11,64 +11,61 @@
 import sys
 from mantid.simpleapi import MoveInstrumentComponent, CropWorkspace
 
-attenuation_correction_pre_2016 = {1.0: 0.007655, 3.0: 1.0, 5.0: 0.005886}
+attenuation_correction_pre_2016 = {1.0: 0.007655, -1.0: 1.0, -1.0: 0.005886}  #values for att_pos 2 and 4 shall not make sense; those attenuators have not been in use that time
 attenuation_correction_post_2016 = {1.0: 1.0, 2.0: 0.00955, 3.0: 0.005886, 4.0: 0.00290, 5.0: 0.00062}
 
-
-#######################################################################################
-# REDUCTION ###########################################################################
-#######################################################################################
-def string_boolean(line):
+####################################################################################### 
+# REDUCTION #######################################################################################
+####################################################################################### 
+def string_boolean(line): 
     """ Convert string to boolean; needed to read "true" and "false" from the csv Data reduction settings table """
-
+    
     if line == 'false':
-        bool_string = False
+       bool_string  = False
     elif line == 'true':
         bool_string = True
-    else:
-        print("Check value of {}".format(line))
-        print("It must be either True or False")
+    else:    
+        print "Check value of ",  line
```

Comment:
for python 3 compatibility, we'll need to keep the print statements as they were and put `from __future__ import print_function` on line 7